js/ui/codeView.js

```diff
@@ -141,7 +141,7 @@ function _synchronizeMetaWindowActorGeometries(src, dst) {
 }
 
 function _synchronizeViewSourceButtonToRectCorner(button, rect) {
-    button.set_position(rect.x - button.width / 2,
+    button.set_position(rect.x,
```

Comment:
Oh! This is probably why the animation didn't work, back when I tried to animate the button...